src/server/config/cmds.go

```diff
@@ -48,6 +48,7 @@ func Cmds() []*cobra.Command {
 	commands = append(commands, cmdutil.CreateAlias(getMetrics, "config get metrics"))
 
 	setMetrics := &cobra.Command{
+		Use:   "{{alias}} (true | false)",
```

Comment:
Nice catch! Were these commands just unusable before?

Reply:
No, they just didn't have the correct usage strings if you messed up or used `--help`.